internal/scheduler/nodedb/nodedb_test.go

```diff
@@ -74,7 +71,7 @@ func TestSelectNodeForPod_NodeIdLabel_Success(t *testing.T) {
 		map[string]string{schedulerconfig.NodeIdLabel: nodeId},
 		testfixtures.N1Cpu4GiJobs("A", testfixtures.PriorityClass0, 1),
 	)
-	jctxs := schedulercontext.JobSchedulingContextsFromJobs(testfixtures.TestPriorityClasses, jobs, func(_ map[string]string) (string, int, int, bool, error) { return "", 1, 1, true, nil })
```

Comment:
I'm glad to be getting rid of this argument to `JobSchedulingContextsFromJobs`: for example, in this case we're returning an empty gang ID and a value of `true` for `isGangJob`, which is illegal.